examples/platform/linux/Options.cpp

```diff
@@ -106,6 +106,9 @@ enum
 #if CHIP_WITH_NLFAULTINJECTION
     kDeviceOption_FaultInjection,
 #endif
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    kDeviceOption_WiFi_PAF = 0x1028,
```

Comment:
```suggestion
    kDeviceOption_WiFi_PAF,
```
Don't assign option values manually